test/IntegrationTests/SmokeTests.cs

```diff
@@ -67,6 +67,20 @@ public async Task WhenStartupHookIsNotEnabled()
 #endif
     }
 
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public async Task WhenClrProfilerIsNotEnabled()
+    {
+        var spans = await RunTestApplicationAsync(enableClrProfiler: false);
+
+#if NETFRAMEWORK
+        // on .NET Framework it is required to set the CLR .NET Profiler
```

Comment:
This comment doesn't match with the `enableClrProfiler: false` from line 74

Reply:
I mean it would be better to say that it doesn't work/instrument on Fx when the profiler is disabled

Reply:
on second reading it may be me...